Source/Fuse.Nodes/VisualBounds.uno

```diff
@@ -307,6 +308,42 @@ namespace Fuse
 
 			return new Box(float3(minX, minY, minZ), float3(maxX, maxY, maxZ));
 		}
+
+		internal enum Type
+		{
+			Render,
+			HitTest
+		}
+
+		internal static VisualBounds Merge(IEnumerable<Visual> visuals, Type bt)
+		{
+			bool hasAnyBounds = false;
+			Box box = new Box(float3(0), float3(0));
+			foreach (var elm in visuals)
```

Comment:
Can you profile this on iOS, I'm still concerend when I see `foreach` in a critical code path (this will be called a lot per frame when something is animated).